include/zenoh-pico/api/types.h

```diff
@@ -338,20 +336,20 @@ typedef _z_reply_data_t z_reply_data_t;
  */
 typedef _z_reply_t z_reply_t;
 
-#define _TYPEDEF_ARRAY(type, alias, elem, name)                               \
-    typedef type alias;                                                       \
-    static inline elem *z_##name##_array_get(const alias *a, size_t k)        \
```

Comment:
I just noticed, could you take Carlos's formatter settings? They should be `"C_Cpp.clang_format_fallbackStyle": "{ BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 120}"` in VSCode's configuration :)